client/src/pages/TaskFormPage/TaskFormPage.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import styles from "./TaskFormPage.module.css";
+import { InputBox } from "../../components/InputBox/InputBox";
+import { CredentialsButton } from "../../components/CredentialsButton/CredentialsButton.js";
+import CloseButton from "react-bootstrap/CloseButton";
+
+const TaskFormPage = () => {
+  return (
+    <div className={styles.bg}>
+      <CloseButton variant="white" />
```

Comment:
Just wondering, is this close button supposed to have a symbol inside of it? (Like an `x`?)
![image](https://user-images.githubusercontent.com/24628243/159596966-72c71fd4-2264-4847-ba6e-dddb796d72d5.png)
Or it is supposed to just be the small rectangle like in the screenshot above?